drape/glfunctions.cpp

```diff
@@ -444,6 +454,13 @@ void GLFunctions::Init()
   glUniform1fvFn = LOAD_GL_FUNC(TglUniform1fvFn, glUniform1fv);
 
   glUniformMatrix4fvFn = LOAD_GL_FUNC(TglUniformMatrix4fvFn, glUniformMatrix4fv);
+
+  /// FBO
```

Comment:
//
or
/// @name FBO
//@{
...
//@}

E.g. either use doxygen notation or do not use it at all.